Gruntfile.js

```diff
@@ -85,21 +82,61 @@ module.exports = function(grunt) {
             }
         },
 
+        // ### grunt-bg-shell
+        // Used to run ember-cli watch in the background
+        bgShell: {
+            ember: {
+                cmd: 'npm run build -- --watch',
+                bg: true,
+                stdout: function (out) {
+                    grunt.log.writeln(chalk.cyan('Ember-cli::') + out);
+                },
+                stderror: function (error) {
+                    grunt.log.error(chalk.red('Ember-cli::' + error));
+                }
+            }
+        },
+
         shell: {
             'npm-install': {
                 command: 'npm install'
             },
 
             'bower-install': {
-                command: path.resolve(cwd + '/node_modules/.bin/bower install')
+                command: 'bower install'
```

Comment:
Does this change require `bower` to be installed globally? The previous method had been used in order to prevent that requirement